package.json

```diff
@@ -50,7 +50,8 @@
     "normalize.css": "^3.0.2",
     "query-string": "^1.0.0",
     "swig": "^1.4.2",
-    "watchify": "^2.0.0"
+    "watchify": "^2.0.0",
+    "metalsmith-i18n": "~0.1.3"
```

Comment:
hmmn not sure of the proper way to uninstall from package.json so I will leave it to you @ahdinosaur :)